java/kafka/pom.xml

```diff
@@ -69,24 +68,29 @@
             <dependency>
                 <groupId>io.opentelemetry</groupId>
                 <artifactId>opentelemetry-api</artifactId>
-                <version>1.17.0</version>
+                <version>${opentelemetry.version}</version>
                 <scope>compile</scope>
             </dependency>
             <dependency>
                 <groupId>io.opentelemetry</groupId>
                 <artifactId>opentelemetry-exporter-jaeger</artifactId>
-                <version>1.17.0</version>
+                <version>${opentelemetry.version}</version>
             </dependency>
             <dependency>
                 <groupId>io.opentelemetry.instrumentation</groupId>
                 <artifactId>opentelemetry-kafka-clients-2.6</artifactId>
-                <version>1.17.0-alpha</version>
+                <version>${opentelemetry-alpha.version}</version>
                 <scope>compile</scope>
             </dependency>
             <dependency>
                 <groupId>io.opentelemetry</groupId>
                 <artifactId>opentelemetry-sdk-extension-autoconfigure</artifactId>
-                <version>1.17.0-alpha</version>
+                <version>${opentelemetry-alpha.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.grpc</groupId>
+                <artifactId>grpc-netty-shaded</artifactId>
+                <version>1.36.1</version>
```

Comment:
Ditto as above, why grpc?

Reply:
This dependency still needs to be removed